make/lib/Awt2dLibraries.gmk

```diff
@@ -433,7 +433,7 @@ $(eval $(call SetupJdkLibrary, BUILD_LIBJAVAJPEG, \
     OPTIMIZATION := HIGHEST, \
     CFLAGS := $(CFLAGS_JDKLIB), \
     HEADERS_FROM_SRC := $(LIBJPEG_HEADERS_FROM_SRC), \
-    DISABLED_WARNINGS_gcc := clobbered implicit-fallthrough shift-negative-value, \
+    DISABLED_WARNINGS_gcc := clobbered implicit-fallthrough shift-negative-value array_bounds, \
```

Comment:
This should be array-bounds. See https://bugs.openjdk.java.net/browse/JDK-8280999

Reply:
Ack. It looks like this fix is already present in 11u-dev https://github.com/openjdk/jdk11u-dev/commit/85c81d9515ba2b98eda9442b3b07c29a230f2a56 so we should pick it up organically. I can cherry-pick that commit into this PR if you like.